packages/analytics-js/src/services/HttpClient/xhr/xhrRequestHandler.ts

```diff
@@ -8,8 +8,8 @@ import type {
 } from '@rudderstack/analytics-js-common/types/HttpClient';
 import type { ILogger } from '@rudderstack/analytics-js-common/types/Logger';
 import { getMutatedError } from '@rudderstack/analytics-js-common/utilities/errors';
+import { FAILED_REQUEST_ERR_MSG_PREFIX } from '@rudderstack/analytics-js-common/constants/errors';
```

Comment:
Does it make sense to move this constant into this module?

Reply:
As per our discussion please resolve it